README.md

```diff
@@ -310,6 +311,20 @@ The default is `:mdns_domain` so that the following remsh invocation works:
 iex --name me@0.0.0.0 --cookie acookie --remsh node_name@nerves.local
 ```
 
+### `:ssh_console_port`
+
+If specified (non-nil), Start an IEx console on a ssh port. This console will
+use the same keys configured with `:nerves_firmware_ssh`.
+You can connect by doing the following:
+```bash
+ssh nerves.local
+```
+
+To exit you will have to do:
+```elixir
+Nerves.InitGadget.SSHConsole.shell_exit()
```

Comment:
Ugh. That's a lot to remember and type. My fingers would like CTRL+D to work, but I'm pretty sure that's a non-starter. Any other ideas for something shorter?

Reply:
I tried ctrl-d and it didn't do anything that i could see. I'm looking right now for a workaround.

Reply:
If we could output some help text for how to exit when a user logs in, that might work. I tried naively wrapping the `Elixir.IEx` module to do this, but I apparently didn't do it right since it didn't work. Do you know of any options for outputting help text on login?

Reply:
How about typing `~.` to quit?